### PR TITLE
Build list of available charts in index

### PR DESCRIPTION
**Pull request by zonca**
Addresses:

https://github.com/jupyterhub/zero-to-jupyterhub-k8s/issues/256

See https://zonca.github.io/helm-chart/

I assume that anything with just one dash is a stable release.

Issues:

* Cannot find a way to list the release dates, the `modified_time` property always has the current time
* I removed 3 very old releases that would make my parsing more complicated

### CODE REVIEW

**Comment by choldgraf** [comment]
really nice! it looks like there is something weird going on with how bullets/indentation are handled:

image

any idea what's going on with that?

**Comment by zonca** [comment]
good catch! a <ul> was missing

**Comment by yuvipanda** [comment]
This is great! The times in 'index.yaml' should be more accurate, since unfortunately the modified times aren't preserved by git clone :( index.yaml is also only accurate because of extra hacks in https://github.com/jupyterhub/zero-to-jupyterhub-k8s/blob/master/build.py.

Ideally, this would also be part of https://github.com/jupyterhub/zero-to-jupyterhub-k8s/blob/master/build.py rather than here via jekyll. We're planning on splitting that script up into its own package soon. However, this PR is strictly an improvement over current status quo, so am happy for this to be merged :)

File: index.md [hunk 1]
```diff
@@ -0,0 +1,50 @@
...
+        {% if chart.extname == '.tgz' %}
+            {% assign chart_splitted = chart.basename | split: "-" %}
+            {% if chart_splitted.size == 2 %}
+            <li> <b><a href="{{ site.url }}/helm-chart{{ chart.path }}"> {{ chart.basename }} </a> </b></li>
+            {% endif %}
+        {% endif %}
+    {% endfor %}
+</ul>
+
+<h2>Development releases: Jupyterhub</h2>
```
Comment (willingc) [inline index.md]: Minor typo: JupyterHub

**Comment by zonca** [comment]
thanks @willingc

**Comment by willingc** [comment]
Thanks @zonca.

I'm going to merge this. Just a heads up to the team @yuvipanda @choldgraf @betatim @minrk @mpacer @jzf2101 @carreau

**Comment by willingc** [comment]
Published at https://jupyterhub.github.io/helm-chart/

Closes https://github.com/jupyterhub/zero-to-jupyterhub-k8s/issues/256

**Pull request merged**